fronts-client/src/types/Collection.ts

```diff
@@ -42,53 +41,54 @@ type NestedCard = NestedCardRootFields & {
   };
 };
 
-interface CardRootMeta extends ChefCardMeta {
-  group?: string;
-  headline?: string;
-  trailText?: string;
-  byline?: string;
-  sportScore?: string;
-  customKicker?: string;
-  href?: string;
-  imageSrc?: string;
-  imageSrcThumb?: string;
-  imageSrcWidth?: string;
-  imageSrcHeight?: string;
-  imageSrcOrigin?: string;
-  imageCutoutSrc?: string;
-  imageCutoutSrcWidth?: string;
-  imageCutoutSrcHeight?: string;
-  imageCutoutSrcOrigin?: string;
-  isBreaking?: boolean;
-  isBoosted?: boolean;
-  showLivePlayable?: boolean;
-  showMainVideo?: boolean;
-  showLargeHeadline?: boolean;
-  showQuotedHeadline?: boolean;
-  showByline?: boolean;
-  imageCutoutReplace?: boolean;
-  imageReplace?: boolean;
-  imageHide?: boolean;
-  showKickerTag?: boolean;
-  showKickerSection?: boolean;
-  showKickerCustom?: boolean;
-  snapUri?: string;
-  snapType?: string;
-  snapCss?: string;
-  atomId?: string;
-  imageSlideshowReplace?: boolean;
-  slideshow?: Array<{
-    src?: string;
-    thumb?: string;
-    width?: string;
-    height?: string;
-    origin?: string;
-  }>;
-  overrideArticleMainMedia?: boolean;
-  coverCardImageReplace?: boolean;
-  coverCardMobileImage?: ImageData;
-  coverCardTabletImage?: ImageData;
-}
+type CardRootMeta = ChefCardMeta &
```

Comment:
Per #1605, the ambition is for the Card model on the client to be a union of different card times, but because this touches a great many places where cards are used, I've opted to avoid this for now to keep the scope of this PR reasonable.